corehq/apps/app_manager/xpath_validator/js/xpathValidator.js

```diff
@@ -0,0 +1,11 @@
+global.SchemeNumber = require('js-xpath/lib/schemeNumber').SchemeNumber;
```

Comment:
Yep, makes sense. I think I missed something crucial when I wrote my comment above. If I understand correctly, this line is fixing a bug in the `js-xpath` and/or `jison` library. In that case, I think what you are doing here is probably a reasonable stopgap measure short of fixing the bug in the offending library. I'd add a comment something like `// HACK expose SchemeNumber global to fix bug in ________` where `________` is filled with the name of the buggy library. Also maybe give a short description of the bug so it can be identified and this line removed if/when it is fixed.